src/sentry/static/sentry/app/components/events/interfaces/richHttpContent/getTransformedData.tsx

```diff
@@ -1,12 +1,18 @@
 import {defined} from 'app/utils';
 
-function getTransformedData(data: any) {
+function getTransformedData(data: any): [string, string][] {
   if (Array.isArray(data)) {
-    return data.filter(dataValue => defined(dataValue));
-  }
-
-  if (typeof data === 'object') {
-    return Object.keys(data).map(key => [key, data[key]]);
+    return data
+      .filter(dataValue => defined(dataValue))
+      .map(dataValue => {
+        if (Array.isArray(dataValue)) {
+          return dataValue;
+        }
+        if (typeof data === 'object') {
+          return Object.keys(dataValue).flatMap(key => [key, dataValue[key]]);
+        }
+        return dataValue;
+      });
   }
```

Comment:
Do we only transform `data` if it's an array then?

Reply:
At first, I thought so - because there can be object inside that array (that's why there were errors in the first place) so I moved it inside, but it turned out that we need to have it in both places.